src/Business Foundation/App/NoSeriesCopilot/src/Copilot/NoSeriesCopilotImpl.Codeunit.al

```diff
@@ -0,0 +1,518 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Foundation.NoSeries;
+
+using System.Telemetry;
+using System.AI;
+using System.Text.Json;
+
+codeunit 324 "No. Series Copilot Impl."
+{
+    var
+        IncorrectCompletionErr: Label 'Incorrect completion. The property %1 is empty', Comment = '%1 = property name';
+        IncorrectCompletionNumberOfGeneratedNoSeriesErr: Label 'Incorrect completion. The number of generated number series is incorrect. Expected %1, but got %2', Comment = '%1 = Expected Number, %2 = Actual Number';
+        TextLengthIsOverMaxLimitErr: Label 'The property %1 exceeds the maximum length of %2', Comment = '%1 = property name, %2 = maximum length';
+        DateSpecificPlaceholderLbl: Label '{current_date}', Locked = true;
+        NotAbleToGenerateNumberSeriesTryToRephraseErr: Label 'Sorry, I am not able to generate the number series. Try to rephrase your request or provide more details.';
+
+    internal procedure GetNoSeriesSuggestions()
+    var
+        FeatureTelemetry: Codeunit "Feature Telemetry";
+        NoSeriesCopilotRegister: Codeunit "No. Series Copilot Register";
+        NoSeriesCopilotImpl: Codeunit "No. Series Copilot Impl.";
```

Comment:
why this variable? as far as i can tell it is only used to get the FeatureName so you could just call this directly